engine/src/main/java/org/terasology/rendering/opengl/GraphicState.java

```diff
@@ -318,7 +345,7 @@ public void preRenderSetupLightGeometry() {
         glDisable(GL_DEPTH_TEST);
 
         glEnable(GL_BLEND);
-        glBlendFunc(GL_ONE, GL_ONE);
+        glBlendFunc(GL_ONE, GL_ONE_MINUS_SRC_COLOR);
```

Comment:
@Josharias: regarding light propagation generating a CPU bottleneck: mental note, it'd be nice to move light propagation to the GPU then, as it is probably parallelizable. That been said, I wonder if it would be horrible to wait until a torch has come to rest before light propagation kicks in. 

@Cervator: I think that's what I was hinting at. But I agree with Josharias this is out of this PR's scope as there would be significant changes.